RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,45 @@
 
 import Cocoa
 
+@objc protocol EntitiesViewDataSource: NSObjectProtocol {
+    optional func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    optional func entitiesView(entitiesView:EntitiesView, titleForEntityAtIndex index:Int) -> String?
+}
+
+@objc protocol EntitiesViewDelegate: NSObjectProtocol {
+    optional func addEntityInEntitiesView(entitiesView:EntitiesView)
+    optional func removeEntityInEntitiesView(entitiesView:EntitiesView, index:Int)
+    optional func entitiesView(entitiesView:EntitiesView, didRemoveEntityAtIndex index:Int)
```

Comment:
add a function for entiteisView(entitiesView:EntitiesView, selectionDidChange index:Int?)